Documents/8-Kigeme-Rwanda-Use-Case.md

```diff
@@ -0,0 +1,239 @@
+![](media/f02e74c3599518984984f5a9dd1d0349.png)
+
+**Humanitarian Engineering for Energy for Displacement (HEED)**
+
+##### Funded by the ESPRC Global Challenges Research Fund
+
+### Renewable Energy Recommendations Tool (RERT) 
+
+**Use Case:** Kigeme Camp Rwanda - Cooking Solutions Evaluation
+
+**Authors:** Sandy Robinson (Scene), Rembrandt Koppelaar (Scene)
```

Comment:
Use a `>` at the beginning of the line to put the formatting in line with the other author sections